src/locales/en-US/commands.json

```diff
@@ -480,13 +480,6 @@
       }
     }
   },
-  "strawpoll": {
-    "commandDescription": "Creates a Straw Poll.",
-    "commandUsage": "<poll title|<option 1|option 2|option 3...>",
-    "noParameters": "You have to give me a title and some options!",
-    "optionsNumberError": "You have to give me a number of options between 2 and 30!",
-    "clickHere": "Click here to vote"
-  },
```

Comment:
Also, i don't remember. Were this command removed? if so, cool note!

Reply:
I couldn't find any other reference to it in the codebase.